views/js/review/provider/qtiTestReviewProvider.js

```diff
@@ -139,6 +140,28 @@ define([
 
             this.assetManager = assetManagerFactory();
 
+            // set tabindex=0 in navigation links and hide other links from navigation
+            $(document).find("li a:not([tabindex])").attr("tabindex", 0);
+            $(document).find("a:not([tabindex])").attr("tabindex", -1);
+            // first and second tab will show block and navigate to panel or content
+            const createJumplink = (link, where) => {
+                if (!link || !where) {
+                    return;
+                }
+                link.on('click', (e) => {
+                    e.preventDefault();
+                    $(e.target).blur();
+                    areaBroker.getArea(where).find(":not(.hidden)[tabindex]").first().focus();
+                });
+                link.on('focus', () => {
+                    areaBroker.getArea(where).addClass('focused');
+                });
+                link.on('blur', () => {
+                    areaBroker.getArea(where).removeClass('focused');
+                });
+            }
+            createJumplink(areaBroker.getContainer().find('.navigationlink'), 'sidebar');
+            createJumplink(areaBroker.getContainer().find('.answerlink'), 'contentWrapper');
```

Comment:
Why didn't you use the `ui/keyNavigation` instead?
Moreover, please avoid to set per element event listeners. Always prefer delegated events.

Reply:
Refactored. Now uses delegated events.